source/_integrations/matter.md

```diff
@@ -23,6 +23,26 @@ ha_integration_type: integration
 
 This integration allows you to communicate with and control Matter devices on your local WiFi or Thread network.
 
+Matter is [the new standard for home automation](https://en.wikipedia.org/wiki/Matter_(standard)) which has just been released and (in the process of) being adopted by the tech industry.
+
+Matter devices are available using either WiFi based communication or [Thread](https://en.wikipedia.org/wiki/Thread_(network_protocol)) and Bluetooth is used for commissioning (adopting) of new devices. Home Assistant supports both transports for Matter.
+
+At this time there are only a few devices available that are compatible with the standard and some of them require you to join a beta/developer program.
+
+It is to be expected that more devices will hit the market during the 2nd quarter of 2023 and beyond.
+
+Both the Matter standard itself and its implementation within Home Assistant are in a very early stage where only the basics are working and/or breaking changes can still happen. Using it in production is not really recommended yet until it matures a bit more.
+
+Matter being a universal standard has some consessions. It is therefore recommended that if an official integration exists (with a local API) within Home Assistant for a specific device, that is recommended over using the Matter protocol to communicate with a device. A good example is Philips Hue where the communication over Matter will only provide the basic controls over lights, the official integration will bring all Hue unique features like (dynamic) scenes, entertainment mode etc.
+
+One of the great features of Matter is the so called "Multi Fabric" feature, which means that you can join the same device to multiple controllers. For example add it to Google Home, Apple Home and Home Assistant at the same time.
+
+In essence, Matter is a local protocol, meaning controlling your devices is done without the need of any cloud. In theory you can purchase a Matter compatible device, join it to Home Assistant (only) and you will have no dependency on an Vender specific cloud or whatsoever.
```

Comment:
Remove all the filler words and words that don't add to the message you're trying to say. Remove "In essence", "In theory", "Meaning"

Also let's move this paragraph to the introduction.